src/coffeeMachine/Main.java

```diff
@@ -1,114 +1,226 @@
 package coffeeMachine;
 
-import java.util.Scanner;
+import java.util.*;
 
 public class Main {
-    private static final int WATER_CONSUMPTION = 200;
-    private static final int MILK_CONSUMPTION = 50;
-    private static final int BEANS_CONSUMPTION = 15;
 
     public static void main(String[] args) {
+      boolean cont = true;
+      VendingMachine m = new VendingMachine();
+      System.out.println(m.getState());
+      String line;
+      Scanner scanner = new Scanner(System.in);
+
+      while (cont) {
+        System.out.print("Write action (buy, fill, take, exit):");
+          try {
+            line = scanner.nextLine().trim();
+              switch (line) {
+              case "buy":
+                  System.out.print("What do you want to buy? 1 - espresso, 2 - latte, 3 - cappuccino:");
+                  line = scanner.nextLine().trim();
+                  switch (line) {
+                    case "1":
+                      m.order(Drink.ESPRESSO);
+                      break;
+
+                    case "2":
+                      m.order(Drink.LATTE);
+                      break;
+
+                    case "3":
+                      m.order(Drink.CAPPUCHINO);
+                      break;
+
+                    default:
+                      System.out.println("Unsupported drink");
+                  }
+                System.out.print(m.getState());
+                System.out.println();
+                break;
+
+              case "fill":
+                  System.out.println("Write how many ml of water do you want to add:");
+                  int water = Integer.parseInt(scanner.nextLine());
+                  System.out.println("Write how many ml of milk do you want to add:");
+                  int milk = Integer.parseInt(scanner.nextLine());
+                  System.out.println("Write how many grams of coffee beans do you want to add:");
+                  int beans = Integer.parseInt(scanner.nextLine());
+                  System.out.println("Write how many disposable cups do you want to add:");
+                  int cups = Integer.parseInt(scanner.nextLine());
+                  m.refill(Refill.getInstance(water,milk,beans,cups));
+                  System.out.print(m.getState());
+                  System.out.println();
+                  break;
+
+              case "take":
+                  int cash = m.take();
+                  System.out.println("I gave you $" + cash);
+                  System.out.print(m.getState());
+                  System.out.println();
+                  break;
+
+              case "exit":
+                  cont = false;
+                  break;
+
+              default:
+                  //System.out.println("Unsupported command");
+            } //eof switch
+          } catch (Exception e) {
+            System.out.println(e.getMessage());
+          }
+      }//eof while
```

Comment:
Is it important to add a comment with eof of a construction? It will be not good in production Java projects. If you want to know about eof after a '}' you may install a plugin for your IDE. Not all members of your team may need to see it.